src/test/java/io/seqera/tower/cli/computeenvs/platforms/GoogleBatchPlatformTest.java

```diff
@@ -74,7 +74,7 @@ void testAddWithAdvancedOptions(MockServerClient mock) throws IOException {
 
         mock.when(
                 request().withMethod("POST").withPath("/compute-envs")
-                        .withBody(JsonBody.json("{\"computeEnv\":{\"name\":\"google\",\"platform\":\"google-batch\",\"config\":{\"location\":\"europe\",\"fusion2Enabled\":true,\"waveEnabled\":true,\"workDir\":\"gs://workdir\",\"usePrivateAddress\":true},\"credentialsId\":\"6XfOhoztUq6de3Dw3X9LSb\"}}")), exactly(1)
+                        .withBody(JsonBody.json("{\"computeEnv\":{\"credentialsId\":\"6XfOhoztUq6de3Dw3X9LSb\",\"name\":\"google\",\"platform\":\"google-batch\",\"config\":{\"location\":\"europe\",\"workDir\":\"gs://workdir\",\"usePrivateAddress\":true,\"waveEnabled\":true,\"fusion2Enabled\":true}}}")), exactly(1)
```

Comment:
There are a lot of changes to the JSON responses in the tests. Is it because of an issue with the sdk update or are they only rearranged for readability?

Reply:
It's because of the update of sdk. Otherwise tests were failing and `mock.openUI()` showed that body didn't match. After changing the order tests passed.

Reply:
oh, so the order of the JSON object fields is relevant? that's annoying